flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/state/rocksdb/RocksDBIncrementalCheckpointUtils.java

```diff
@@ -115,7 +115,6 @@ public double getOverlapFraction() {
         public int compareTo(@Nullable Score other) {
             return Comparator.nullsFirst(
                             Comparator.comparing(Score::getIntersectGroupRange)
-                                    .thenComparing(Score::getIntersectGroupRange)
```

Comment:
Please could you add a unit test for this scenario

Reply:
Ah I see - it is not change behaviour just removing the duplicate. I agree no need to unit test.